app/src/db/migrations/20240327213908_bring-forward-feat.ts

```diff
@@ -0,0 +1,31 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return Promise.resolve()
+    .then(() =>
+      knex.schema.alterTable('submission', function (table) {
+        table.dropColumn('bring_forward_date');
+      })
+    )
+    .then(() =>
+      knex.schema.alterTable('note', function (table) {
+        table.timestamp('bring_forward_date', { useTz: true });
+        table.text('bring_forward_state').defaultTo('');
```

Comment:
Can we make this nullable instead of defaulting to an empty string. NULL is a better representation of no value.